recipe/meta.yaml

```diff
@@ -68,18 +73,22 @@ outputs:
         - test -f ${PREFIX}/lib/libLLVMSPIRVLib.so.{{ abi_version }}     # [linux]
 
   - name: libllvmspirv
-    script: install.sh
+    script: install.sh   # [not win]
+    script: install.bat  # [win]
     build:
       run_exports:
         - {{ pin_subpackage("libllvmspirv" ~ major) }}
```

Comment:
This need to be done only for `# [not win]`

Reply:
Thank you! I've missed that one